src/pytorch_lightning/strategies/ddp.py

```diff
@@ -24,7 +24,6 @@
 import torch
 import torch.distributed
 from torch import Tensor
-from torch.distributed.constants import default_pg_timeout
```

Comment:
Can you keep this under `is_available` as you did everywhere else, and fallback to `timedelta` otherwise?

Reply:
Yes, it makes much sense.